features/step_definitions/steps.rb

```diff
@@ -1,6 +1,9 @@
+Given("I am not signed in") do
```

Comment:
Probably should add that this is an intentional NOOP here.

Reply:
I like this -- emphasizes that the Gherkin steps are about establishing context for the people who are writing/reading the stories, rather than about using Gherkin to create a feature testing DSL.